source/includes/api/_before_you_get_started.md

```diff
@@ -56,6 +56,130 @@ Below you can find a list of RW API resources that use the `application` field:
+
+Multiple sorting criteria can be used, separating them by commas. You can also specify the sorting order by prepending the criteria with either `-` for descending order or `+` for ascending order. By default, ascending order is assumed.
+
+Keep in mind that it’s up to each individual RW API service (dataset, widget, layer, etc) to define and implement the sorting mechanisms. Because of this, the examples above may not be true for all cases. Refer to the documentation of each resource and endpoint for more details on sorting.
+
+### Which services comply with these guidelines
+
+The following endpoints adhere to the Sorting conventions defined above:
+
+* [Get v2 areas endpoint](/index-rw.html#getting-all-user-areas)
+* [Get areas endpoint](/index-rw.html#get-user-areas)
+* [Get collections endpoint](/index-rw.html#getting-collections-for-the-request-user)
+* [Get dashboards endpoint](/index-rw.html#getting-all-dashboards)
+* [Get datasets endpoint](/index-rw.html#getting-all-datasets)
+* [Get layers endpoint](/index-rw.html#getting-all-layers)
+* [Get metadata endpoint](/index-rw.html#getting-all-metadata)
+* [Get widgets endpoint](/index-rw.html#getting-all-widgets)
+
+## Filtering
+
+> Example request filtering using a single condition:
+
+```shell
+curl -X GET https://api.resourcewatch.org/v1/dataset?name=viirs
+```
+
+> Example request filtering using multiple conditions:
+
+```shell
+curl -X GET https://api.resourcewatch.org/v1/dataset?name=birds&provider=cartodb
+```
+
+> Example request filtering by an array field using the `,` OR multi-value separator:
+
+```shell
+curl -X GET https://api.resourcewatch.org/v1/dataset?application=rw,gfw
+```
+
+> Example request filtering by an array field using the `@` AND multi-value separator:
+
+```shell
+curl -X GET https://api.resourcewatch.org/v1/dataset?application=rw@gfw
+```
+
+Like in the case of sorting, most RW API resources allow filtering the returned results of list endpoints using query parameters. As a rule of thumb, you can use the API resource's fields as query parameter filters, as shown in the examples on the side. You can also combine different query parameters into a complex `and` logic filter.
+
+String type fields support and expect a regex value, unless detailed otherwise on documentation for each API resource. Although typically they will match exact strings, you may have to escape certain characters (PCRE v8.42 spec).
```

Comment:
This is a little confusing. So is it a regex value or an exact string?

Reply:
It's complicated. All string filters are passed to MongoDB as case-insensitive regexes: https://github.com/resource-watch/dataset/blob/a0ec9cd728df600f1209f7fb0eee90b65cfb35d3/src/services/dataset.service.js#L128

However, from my experiments, it seems that MongoDB is "smart" enough to produce the results you would expect if you were to use this filter as a string, rather than a regex. That, of course, will come with its flaws, so we can't really promise it will work for all strings as-is - hence the "you may need to escape stuff". 

So, long story short, I see two paths:
- we tell users that they may be able to get away with simple strings in most cases, but that they may have to deal/may be able to use regexes in some cases. This is the current text
- we tell users "always use regexes", which is truer to spec, and arguably the safer approach for us, but also is technically a more complex solution (I don't know about you, but the word "regexes" induces some anxiety in me).

Reply:
How about this?

```suggestion
For string type fields, the filter you pass will be interpreted as a regular expression, _not_ as a simple substring filter. This gives you greater flexibility in your search capabilities. However, it means that, if you intend to search by substring, you must escape any regex special characters in the string.
```